.freeCodeCamp/tooling/test-utils.js

```diff
@@ -257,6 +238,11 @@ async function startSocketServerAndHandshake({
       });
     });
 
+    setTimeout(() => {
+      server.close();
+      resolve();
+    }, 5000);
```

Comment:
How did you decide 5 seconds?

Reply:
randomly - I dunno. If their project is working, it should only take a fraction of second for everything to run and pass. Five seconds seemed like a good buffer. Do you have any other suggestions for it?